config/locales/en.yml

```
@@ -1006,6 +1048,15 @@ en:
       remove_choice: Remove Choice
     CurriculumEditor__TargetDetailsEditor:
       add_another_question: Add another question
+      assignment_discussion:
+        help: Enabling discussions allow students to see and engage in discussions of other student submissions
+        help_url: https://docs.pupilfirst.com/users/curriculum_editor#enable-discussion
+        label: Do you want to enable discussion on this assignment?
+      allow_anonymous:
+        title: Setup submission anonymity
+        sub_title: Select whether or not a student can choose to submit assignment anonymously
```

Stick to either singular or plural reference. We tend to use the plural when referring to the group.

```suggestion
        sub_title: Select whether or not students can choose to submit assignment anonymously
```